init.sql

```diff
@@ -3,6 +3,11 @@
 -- NOTE: The role lemmingpants and the database lemmingpants must exist!
 --
 -- Drop all schemas, recreate them and make sure that Lemmingpants own them.
+ALTER DATABASE lemmingpants SET "app.jwt_secret" TO "feBU1ykZ4icKs2nKam9l8CD84qhgeOl6QQakrUJBiRTUu4dKTLVoH8o";
+CREATE EXTENSION pgcrypto;
```

Comment:
Add an `IF NOT EXISTS` to all `CREATE EXTENSION` make them more robust. Because on Hackeholken or a local dev machine that doesn't run Docker, a superuser will already have installed all the extensions.

Reference: https://www.postgresql.org/docs/9.6/static/sql-createextension.html